src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
```

Comment:
Consider using a more specific class name instead of `.PostInfo`. This will help avoid potential conflicts with other components or styles. 🎨